RFCs/RFC-0005: Protocol Upgrade to Enable Automatic Distribution of Delegation Rewards.md

```diff
@@ -0,0 +1,49 @@
+# RFC-0005: [Protocol Upgrade to Enable Automatic Distribution of Delegation Rewards]
+
+- **Intent**: [To request a protocol upgrade for automatic distribution of delegation rewards.]
+- **Submitted by**: [lamps, discord: lamps6718, twitter:lamps945 ]
+- **Submitted on**: [26 Dec 2023]
+
+## Abstract
+
+This RFC aims to address one big pain point affecting all node operators on Mina network, which is the distribution of delegation rewards. The current mechanism relies the node operators to voluntarily calculate the rewards and send them back to the delegators. It is error-prone for the calculations and, more importantly, relies complete trust of the node operator, which hinders the decentralisation of the network. The RFC requests a redesign of the mechanism and achieve one that is similar to the Cosmos chains of Cardano, where the rewards are not accessible to the node operators, and the delegators can check and claim the rewards any time.   
```

Comment:
> The RFC requests a redesign of the mechanism

Core RFCs are not for requesting artefacts like a "redesign" directly but rather to conjure up artefacts that allow one to produce such an artefact by outlining scenarios, and requirements as well as engaging in discussion to refine those items. Please reword this last sentence 🙏